cppForSwig/BlockUtils.cpp

```diff
@@ -1022,6 +1022,15 @@ void BlockDataManagerConfig::parseArgs(int argc, char* argv[])
          dbDir_ = move(newPath);
       }
 
+      if (access(dbDir_.c_str(), F_OK | R_OK | W_OK) != 0)
+      {
+#ifdef _WIN32
+         CreateDirectory(dbDir_.c_str());
```

Comment:
Need to check for errors in both functions.

This blind creation will hide bugs.